profile/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center">
+<img src="https://electricracing.northeastern.edu/wp-content/uploads/2023/02/cropped-NER-logo-white.png" alt="drawing" width="200"/>
+</p>
+
+<p align="center">
+<a href="https://electricracing.northeastern.edu">Website </a> | <a href="https://finishlinebyner.com">FinishLine </a> | <a href="https://www.linkedin.com/company/northeasternelectricracing"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/LinkedIn.svg" width="16" height="16"> </a>
```

Comment:
We do have a FinishLine logo we can use, and then Id wrap the NER logo in a link and have it link to our general website